src/main/resources/scripts/furnace automatisation.sk

```diff
@@ -12,32 +12,29 @@ options:
 	#   1: only chests direcly next to the furnace are affected
 	#   1.5: chests diaginally adjacent (i.e. which touch the furnace with one edge) are affected as well
 	#   1.75: all surrounding chests in a 3x3x3 cube are affected
-	radius: 1
+	radius: 2
 
 on fuel burn:
-	fuel slot of block will be empty
-	loop blocks in radius {@radius}:
-		loop-block is chest
-		loop items of type fuel:
-			loop-block contains loop-item
-			remove loop-item from loop-block
-			set fuel of event-block to loop-item
-			stop trigger
+	if fuel slot of block will be empty:
+		loop blocks in radius {@radius}:
+			if loop-block is a chest:

```

Comment:
the old format was better in this specific case, using in-line in this kind of case is important.

Reply:
This script doesn't work properly anyway, I didn't really attempt fixing it either, I'll look into it and revert to the old format.

Reply:
I've tried fixing this with @Pikachu920 and we can't get it to set the fuel in the furnace, so idk what to do.